qtribu/gui/dck_qchat.py

```diff
@@ -1033,6 +1033,13 @@ def on_send_layer_to_qchat(self) -> None:
         exporter.setDestinationCrs(layer.crs())
         exporter.setTransformGeometries(True)
         geojson_str = exporter.exportFeatures(layer.getFeatures())
+
+        # save and read QML style to and from temp file
+        save_style_path = os.path.join(tempfile.gettempdir(), "qchat_layer_style.qml")
+        layer.saveNamedStyle(save_style_path, categories=QgsMapLayer.AllStyleCategories)
+        with open(save_style_path, "r") as file:
```

Comment:
You should enforce encoding and catch exceptions. Our frienddly end-users are mainly on Windows and who knows which encoding table they use!